neo/SmartContract/Native/Votes/Method/FindAllPathMethod.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract.Native
+{
+    public class FindAllPathMethod
+    {
+        /// <summary>
+        /// Node Set
+        /// </summary>
+        private List<String> nodesList = new List<String>();
+        /// <summary>
+        /// Vector Graphic,describe the connection between nodes and nodes
+        /// </summary>
+        private Dictionary<String, int> vectorGraphic = new Dictionary<String, int>();
+        /// <summary>
+        /// Edge Array,describe the connection between nodes and nodes
+        /// </summary>
+        private int[,] edgeArray;
+        /// <summary>
+        /// Path Set,store paths in vector graphic.
+        /// </summary>
+        private List<List<String>> path = new List<List<String>>();
+
+        public FindAllPathMethod(List<String> nodesList, Dictionary<String, int> vectorGraphic)
+        {
+            this.nodesList = nodesList;
+            this.vectorGraphic = vectorGraphic;
+            CreateEdgeArray();
+        }
+
+        /// <summary>
+        /// create edge array
+        /// </summary>
+        private void CreateEdgeArray()
+        {
+            int nodeCount = nodesList.Count;
+            edgeArray = new int[nodeCount, nodeCount];
+            for (int i = 0; i < nodeCount; i++)
+                for (int j = 0; j < nodeCount; j++)
+                {
+                    edgeArray[i, j] = -1;
```

Comment:
I think that there should be a way to initialize all matrix with -1 in C# in a single line, but not sure.
Otherwise, maybe let's create a class for this int[,] as we described above. This class would handle this initialization and other operations.

Reply:
maybe something like 
```C#
Enumerable.Repeat(-1,m*n)
  .Select(d => Enumerable.Repeat(0,m).ToArrray())
  .ToArray();
```

but still a bit different.

HAHA

Reply:
Actually `int[,]` is different from `int[][]` and `int[,]` may be a bit more efficient

Reply:
Mabye it is a little difficult.I can't  find a function to do this.This is not a C# guy here. hahaha